Assets/Countly/Plugins/CountlySDK/Services/UserDetailsCountlyService.cs

```diff
@@ -24,54 +23,188 @@ internal UserDetailsCountlyService(CountlyConfiguration configuration, CountlyLo
             CustomDataProperties = new Dictionary<string, object>();
         }
 
+        #region PublicAPI
         /// <summary>
-        /// Add user custom detail to request queue.
+        /// Sets information about user.
```

Comment:
does it set name or etc. func comment might not address the correct intention of the function

Reply:
this definetly is a copy paste error. fixing it